server/src/main/scala/io/delta/standalone/internal/DeltaSharedTableLoader.scala

```diff
@@ -288,6 +367,13 @@ class DeltaSharedTable(
     deltaLog.update()
   }
 
+  private def protocolRead(protocol: Protocol): Unit = {
```

Comment:
Should this be called: validateProtocol ?

Reply:
updated as `assertProtocolRead` to be consistent with this: https://github.com/delta-io/connectors/blob/master/standalone/src/main/scala/io/delta/standalone/internal/DeltaLogImpl.scala#L212